src/evaluate/evaluator/token_classification.py

```diff
@@ -94,8 +94,19 @@ def words_to_offsets(self, words: List[str], join_by: str):
 
         return offsets
 
-    def prepare_data(self, data: Union[str, Dataset], input_column: str, label_column: str, join_by: str):
-        super().prepare_data(data, input_column, label_column)
+    # def postprocess_data(self, data):
+    #
+    #     return metric_inputs, pipeline_inputs
+    def prepare_data(
+        self, data: Union[str, Dataset], input_column: str, label_column: str, join_by: str, data_split: Optional[str]
+    ):
+        # super().prepare_data(data, input_column, label_column, data_split)
```

Comment:
I'm unclear on the original reason for this — was it to avoid code duplication?

Reply:
yes that was the original motivation - we want to carry out these checks https://github.com/huggingface/evaluate/blob/main/src/evaluate/evaluator/base.py#L240-L247 - It makes sense to remove the call to the parent class, but let's copy the checks here